cmd/thanos/tools_bucket.go

```diff
@@ -291,6 +325,7 @@ func registerBucket(app extkingpin.AppClause) {
 	registerBucketMarkBlock(cmd, objStoreConfig)
 	registerBucketRewrite(cmd, objStoreConfig)
 	registerBucketRetention(cmd, objStoreConfig)
+	registerBucketUploadSnapshotBlocks(cmd, objStoreConfig)
```

Comment:
Umm, can we generalize the tool to upload general blocks? Why does it needs to be specifically snapshot blocks